board/embeddedartists/mx7ulpea-ucom/mx7ulpea-ucom.c

```diff
@@ -95,7 +298,6 @@ int board_init(void)
 #ifdef CONFIG_FSL_QSPI
 	board_qspi_init();
 #endif
-
```

Comment:
Do we need this line removed?

Reply:
Had the same comment, @ldts mind removing the removal?